cpp/src/barretenberg/crypto/schnorr/c_bind.cpp

```diff
@@ -80,8 +80,8 @@ WASM_EXPORT void multisig_construct_signature_round_1(uint8_t* round_one_public_
     using multisig = crypto::schnorr::multisig<grumpkin::g1, KeccakHasher, Blake2sHasher>;
 
     auto [public_output, private_output] = multisig::construct_signature_round_1();
-    write(round_one_public_output_buf, public_output);
-    write(round_one_private_output_buf, private_output);
+    serialize::write_msgpack(round_one_public_output_buf, public_output);
+    serialize::write(round_one_private_output_buf, private_output);
```

Comment:
Why write_msgpack for one and write for the other?

Reply:
I'll try undoing this hack now, thanks for the reminder